addons/mail/static/src/core/message_model.js

```diff
@@ -181,7 +181,7 @@ export class Message {
     }
 
     get hasTextContent() {
-        return (this.editDate && this.attachments.length) || !this.isBodyEmpty;
+        return /*(this.editDate && this.attachments.length) || */ !this.isBodyEmpty;
```

Comment:
remove this comment

Reply:
The feature of edit date was not fully removed in 16.3. We just hide the edited label.
This is the minimal diff to disable it for the time being and fix it once and for all in master.
The less diff the better, and sooner or later we will need to add it back, as the JS is fine but the python needs to change (i.e. not use `create_date` / `write_date`)